Client/Frontend/Brave Rewards/OptIn/BraveTalkRewardsOptInView.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveUI
+import Shared
+import BraveShared
+
+extension BraveTalkRewardsOptInViewController {
+    class View: UIView {
+        
+        let enableRewardsButton = ActionButton().then {
+            $0.layer.borderWidth = 0
+            $0.titleLabel?.font = .systemFont(ofSize: 16.0, weight: .semibold)
+            $0.setTitleColor(.white, for: .normal)
+            $0.setTitle(Strings.Rewards.braveTalkRewardsOptInButtonTitle, for: .normal)
+            $0.contentEdgeInsets = UIEdgeInsets(top: 10, left: 20, bottom: 10, right: 20)
+            $0.backgroundColor = .braveLighterBlurple
+        }
+        
+        private let image = UIImageView(image: #imageLiteral(resourceName: "rewards_onboarding_cashback")).then {
+            $0.contentMode = .scaleAspectFit
+            
+            $0.layer.shadowColor = UIColor.black.cgColor
+            $0.layer.shadowOpacity = 0.25
+            $0.layer.shadowOffset = CGSize(width: 0, height: 1)
+            $0.layer.shadowRadius = 4
+        }
+        
+        private let title = UILabel().then {
+            $0.text = Strings.Rewards.braveTalkRewardsOptInTitle
+            $0.font = .systemFont(ofSize: 20)
```

Comment:
Use preferred fonts/scaled fonts and set `adjustsFontForContentSizeCategory` to true